### DISCUSSION TITLE
Not able to Login to ente auth

### DISCUSSION THREAD

**Discussion by Xfire08** [ente-io/ente#3084] [Q&A]
I have just done format of laptop and have installed windows 10 (earlier was using win11 ) and I am trying to login it just doesnt work , I am able to login via web , attaching logs.
[ente-logs-2024-Sep-1-9-40.zip](https://github.com/user-attachments/files/16828377/ente-logs-2024-Sep-1-9-40.zip)

**Comment by Xfire08** [comment]
Let me know if I need to do anything different to make it work

**Reply by ua741** [reply]
Can you please follow these steps, and let us know if it helps https://help.ente.io/auth/troubleshooting/windows-login

**Comment by Xfire08** [comment]
@ua741  I have moved to other authenticator app and hence wont be able to perform these steps .
